dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/MultipleApproverActionlet.java

```diff
@@ -1,30 +1,21 @@
 package com.dotmarketing.portlets.workflows.actionlet;
 
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.StringTokenizer;
-
 import com.dotmarketing.business.APILocator;
 import com.dotmarketing.exception.DotDataException;
-import com.dotmarketing.portlets.workflows.model.MultiEmailParameter;
-import com.dotmarketing.portlets.workflows.model.WorkflowActionClassParameter;
-import com.dotmarketing.portlets.workflows.model.WorkflowActionFailureException;
-import com.dotmarketing.portlets.workflows.model.WorkflowActionletParameter;
-import com.dotmarketing.portlets.workflows.model.WorkflowHistory;
-import com.dotmarketing.portlets.workflows.model.WorkflowProcessor;
+import com.dotmarketing.portlets.workflows.model.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'com.dotmarketing.portlets.workflows.model'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4904110)

Reply:
I think wildcard imports are not desirable in large code bases as they pollute the namespace and can cause compilation issues in the future. We need to agree on a code style/formatter for intellij and all use it to avoid changes like this.